Lib/test/test_capi.py

```diff
@@ -311,7 +311,7 @@ def test_getitem_with_error(self):
                     r'Current thread .* \(most recent call first\):\n'
                     r'  File .*, line 6 in <module>\n'
                     r'\n'
-                    r'Extension modules: _testcapi \(total: 1\)\n')
+                    r'Extension modules: .* \(total: 1\)\n')
```

Comment:
Why this change?

Reply:
Since `_copy` is now a C extension it is reported in the traceback. (before `_testcapi` was the only module)